### PR TITLE
add jscs and mofo-style, npm script

### PR DESCRIPTION
**Pull request by ashleygwilliams**
solves #25

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
so jshint barfs about const. how do we feel about this? cc/ @simonwex

**Comment by simonwex** [comment]
Happy to move forward with ES6 in our style-guide. Timing might be an issue as @gvn and @Pomax who care greatly about this stuff are in :shipit: <(SHIP IT) mode.

**Comment by sedge** [comment]
Shall we pull out `const` for now then?

File: package.json [hunk 1]
```diff
@@ -7,7 +7,10 @@
     "start": "node server",
     "env": "cp env.dist .env",
     "db": "knex migrate:latest && knex seed:run",
-    "test": "lab"
+    "test": "lab",
```
Comment (gideonthomas) [inline package.json]: I think we should add `npm run lint` to the test script as well

Comment (sedge) [inline package.json]: I'll add it in #32

Comment (ashleygwilliams) [inline package.json]: :+1:

**Comment by ashleygwilliams** [comment]
ok, playing nice with mofo-style. final review appreciated :bow:

**Comment by gideonthomas** [comment]
r=me

**Comment by ashleygwilliams** [comment]
ok squashing

**Pull request merged**